pkg/auth/jwt.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020-present Open Networking Foundation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"google.golang.org/grpc/codes"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+const (
+	// SharedSecretKey shared secret key for signing a token
+	SharedSecretKey = "SHARED_SECRET_KEY"
+)
+
+// JwtAuthenticator jwt authenticator
+type JwtAuthenticator struct {
+}
+
+// NewJwtAuthenticator create an instance of jwt authenticator
+func NewJwtAuthenticator() *JwtAuthenticator {
```

Comment:
I feel this kind of thing is not required, unless `JwtAuthenticator` was private i.e. `jwtAuthenticator`
a caller could call `new(JwtAuthenticator)` and get the same result

Reply:
At the moment, yes that is true and we can remove it but I was not sure if at some point we need to add some info about keys into the struct and will be initialized when we want to use the package.